### PR TITLE
feat: add fix benchmark

### PR DESCRIPTION
**Pull request by a4blue**
Added a Benchmark for the fix Function
#389 

#### ✔ Checklist:

- [x] Commit messages have been written in [Conventional Commits](https://www.conventionalcommits.org) format;
- [x] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [ ] Docs have been added / updated on the [dotenv-linter.github.io](https://github.com/dotenv-linter/dotenv-linter.github.io) (for bug fixes / features).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,3 +12,4 @@
 *.env
 *.env_*
 !/tests/fixes/fixtures/*.env
+!/benches/fixtures/*.env
```
Comment (mgrachev) [inline .gitignore:15]: Why these changes is needed?

File: benches/fix.rs [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                let temp = tempdir().expect("create tempdir");
+                let path = temp.into_path();
+                let simple_fix_path = path.join("simple_fix.env");
+
+                let matches = app.clone().get_matches_from(vec![
+                    "dotenv-linter",
+                    // Flag doesn't work like this
+                    // "no-backup",
+                    simple_fix_path.to_str().expect("path to str"),
+                ]);
```
Comment (mgrachev) [inline benches/fix.rs:27]:
```suggestion
                let matches = app.clone().get_matches_from(vec![
                    "dotenv-linter",
                    "fix",
                    "--no-backup",
                    simple_fix_path.to_str().expect("path to str"),
                ]);
                let matches = matches
                    .subcommand_matches("fix")
                    .expect("fix command")
                    .to_owned();
```
Try this code 😉

**Review by mgrachev — COMMENTED** [review]
@a4blue 👋 Thank you for your contribution 💪

I have left some comments. Please take a look at them 👀
Also, can you add another benchmark with backups?

File: .gitignore (hunk 1 above)
Comment (a4blue) [reply .gitignore]: Was a mistake, thanks for reporting this :)

File: benches/fix.rs (hunk 2 above)
Comment (a4blue) [reply benches/fix.rs]: That fixed it 😄

**Comment by a4blue** [comment]
Fix Benchmarks with and without Backups should be implemented now.
Did a small refactor in order to minimize Code Duplication.
I am however not so sure if i understood the lifetimes of the newly written function `generate_arg_matches` well enough 😟

**Comment by mgrachev** [comment]
@a4blue Please resolve conflicts.

File: .gitignore [hunk 3]
```diff
@@ -11,4 +11,4 @@
 # Environment Files
 *.env
 *.env_*
-!/tests/fixes/fixtures/*.env
+!/tests/fixes/fixtures/*.env
```
Comment (DDtKey) [inline .gitignore:14]: Please, add line break at the end of file 🙂

**Review by DDtKey — APPROVED** [review]

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@a4blue Thank you for your help 👍

🙏 If it’s not difficult for you, please support the project - star on GitHub ⭐️
